static/app/views/performance/newTraceDetails/traceRenderers/virtualizedViewManager.tsx

```diff
@@ -11,7 +11,9 @@ import type {
 } from 'sentry/views/performance/newTraceDetails/traceModels/traceTree';
```

Comment:
A lot of the changes from this file is just code that was moved to TraceView and updates to access the view on the stored field